spec/serializer_spec.rb

```diff
@@ -350,6 +354,65 @@ def serialize_primary(object, options = {})
     end
   end
 
+  # The members data and errors MUST NOT coexist in the same document.
+  describe 'JSONAPI::Serializer.serialize_errors' do
+    it 'can include a top level errors node' do
+      errors = [
+        {
+          "source" => { "pointer" => "/data/attributes/first-name" },
```

Comment:
Super nitpick: can you replace all the double quotes everywhere with single quotes? And also remove the whitespace inside these hashes, like `{"pointer" => "/data/attributes/first-name"}` instead. I swear I will set up a linter soon. :)